src/main/java/frc/robot/subsystems/arm/ArmStageTwo.java

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.arm;
+
+import com.ctre.phoenix.motorcontrol.FeedbackDevice;
+import com.ctre.phoenix.motorcontrol.InvertType;
+import com.ctre.phoenix.motorcontrol.TalonSRXControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.revrobotics.*;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj.DutyCycleEncoder;
+import edu.wpi.first.wpilibj.motorcontrol.PWMSparkMax;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+import static frc.robot.Constants.ArmConstants;
+
+public class ArmStageTwo extends SubsystemBase {
+  private CANSparkMax armMotorPrimary;
+  private CANSparkMax armMotorSecondary;
+  private RelativeEncoder encoder;
+  private SparkMaxPIDController pidController;
+
+  private double encoderOffset = ArmConstants.kStageTwo_AbsEncoderInitialOffset;
```

Comment:
should we not just be using an initial position? I feel like the offset thing just makes me think more, but maybe there is a reason

Reply:
It felt logical at the time but I think I was definitely just overcomplicating things